src/test/groovy/org/mockito/release/notes/vcs/IgnoredCommitTest.groovy

```diff
@@ -0,0 +1,24 @@
+package org.mockito.release.notes.vcs
+
+import spock.lang.Specification
+
+class IgnoredCommitTest extends Specification {
+
+    def "should skip commits with that contains given ignored substrings"() {
+        def ignoredCommit = new IgnoredCommit(["[ci skip]", "[custom skip]"])
+        def commitWithCiSkip = new GitCommit("firstCommitId", "sample@email.com", "sampleAuthor", "sample [ci skip] commit message")
+        def commitWithCustomSkip = new GitCommit("thirdCommitId", "sample@email.com", "sampleAuthor", "sample [custom skip] commit message")
+
+        expect:
+        ignoredCommit.isTrue(commitWithCiSkip)
+        ignoredCommit.isTrue(commitWithCustomSkip)
+    }
+
+    def "should not skip commit that does not contain given ignored substrings"() {
+        def ignoredCommit = new IgnoredCommit(["[ci skip]", "[custom skip]"])
+        def commit = new GitCommit("firstCommitId", "sample@email.com", "sampleAuthor", "sample commit message")
```

Comment:
It's a good idea if GitCommit is created also in other places. However in this particular case the simplest solution would be to create a private method, that would instantiate a GitCommit. Since only message is important, you can randomly generate other properties, see RandomStringUtils from commons-lang, it's already on classpath.